### ISSUE TITLE
find_files has a nonsensical error message when called with wrong arity

### ISSUE THREAD

**Issue by jmgao** [jimporter/bfg9000#93]
```
error: build.bfg:1: unknown language "None"
  File "build.bfg", line 1, in <module>
    executable("foo", files=find_files("foo"))
```

**Comment by jmgao** [comment]
Why not just glob all of the paths, anyway?

**Comment by jimporter** [comment]
I think this is actually an issue with the `object_file` function, not `find_files`.

> Why not just glob all of the paths, anyway?

Not sure what you mean by this. Do you mean you want to implicitly run `find_files` whenever there's a `"*"` or whatever?

**Comment by jimporter** [comment]
Resolved by 703a18faf64cee3513e0749ca7ee681ab035d2d2

**Comment by jmgao** [comment]
> > Why not just glob all of the paths, anyway?
> 
> Not sure what you mean by this. Do you mean you want to implicitly run `find_files` whenever there's a `"*"` or whatever?

I don't remember if we talked about this in IRC, but what I meant by this was probably "why doesn't this support something along the lines of `executable("foo", files=["foo.cpp", "bar/*.cpp"])`, running glob on each element of the array?"
